blockchain/accept.go

```diff
@@ -199,6 +199,20 @@ func ticketsRevokedInBlock(bl *dcrutil.Block) []chainhash.Hash {
 	return tickets
 }
 
+// voteVersionsInBlock returns all versions in a block.
+func voteVersionsInBlock(bl *dcrutil.Block) []uint32 {
+	versions := make([]uint32, 0, 5)
```

Comment:
5 --> pass params *chaincfg.Params, then use params.TicketsPerBlock. params is found in the blockchain structure.